### DISCUSSION TITLE
Télémagino [CA]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#920] [Q&A]
### Channel ID

Telemagino.ca

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
cable channel  will be hard to have it legaly

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19860
